test/test_jit_cuda_fuser.py

```diff
@@ -4,14 +4,20 @@
 
 import torch
 
-from torch.testing._internal.common_utils import run_tests, ProfilingMode, GRAPH_EXECUTOR
+from torch.testing._internal.common_utils import run_tests, ProfilingMode, GRAPH_EXECUTOR, UNITTEST_ARGS
 from torch.testing._internal.codegen.random_topo_test import runDefaultTestWithSeed
 
 from test_jit import JitTestCase, RUN_CUDA
 import itertools
 import numpy as np
+import math
+
+if '-nvfuser_do_fallback' in UNITTEST_ARGS:
+    NVFUSER_DISABLE_FALLBACK = False
```

Comment:
We should set `os.environ['PYTORCH_NVFUSER_DISABLE_FALLBACK'] = '0'` in case it's turned off somehow, since the arguments are explicitly enabling fallback.